dataland-internal-storage/src/test/resources/db/migration/V12/expectedDatabaseEntry.json

```diff
@@ -0,0 +1,33 @@
+{
+  "general": {
+    "general": {
+      "data-date": "2023-11-23",
+      "fiscalYearDeviation": "Deviation",
+      "fiscalYearEnd": "2022-31-12"
+    }
+  },
+  "environmental": {
+    "energyPerformance": {
+      "applicableHighImpactClimateSectors": {
+        "NaceCodeA": {
+          "highImpactClimateSectorEnergyConsumptionInGWh": {
```

Comment:
Is there something to do for the new field InGWhPerMioEuro?

Reply:
I talked to Siegfried about this and completely new fields do not need any migration effort for being interpreted as null after migration

Reply:
Ok